hbase-shell/src/main/ruby/shell/commands/get_slowlog_responses.rb

```diff
@@ -0,0 +1,65 @@
+#
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with this
+# work for additional information regarding copyright ownership. The ASF
+# licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Retrieve latest slowlog responses maintained in memory by RegionServers
+
+module Shell
+  module Commands
+    # Retrieve latest slowlog responses
+    class GetSlowlogResponses < Command
+      def help
+        <<-EOF
+Retrieve latest SlowLog Responses maintained by each or specific RegionServers.
+Specify 'all_rs' to include all RS otherwise array of 'server_name' for specific
+RS. 'server_name' is the host, port plus startcode of a RegionServer.
```

Comment:
Yeah, use '*' instead of 'all_rs'?  And for server_name, is this the ServerName ? See other commands for how they discuss passing ServerName because I don't think this aligns; e.g. no where else to we talk of 'server_name' IIRC

Reply:
I think we always pass server_name as `host:port:startcode` of a RegionServer right? That is what would work even here.

Reply:
Address both points